### ISSUE TITLE
Boxes.py

### ISSUE THREAD

**Issue by hellerbarde** [hellerbarde/inkscape-boxmaker#6]
Look into merging this project with [Boxes.py](https://github.com/florianfesti/boxes). 

Try it [here](https://www.festi.info/boxes.py/)

let's see :)

**Comment by hellerbarde** [comment]
@roastedneutrons Do you have any interest in helping with this?

**Comment by hellerbarde** [comment]
actually, they are a inkscape extension already too.

**Comment by roastedneutrons** [comment]
It looks great. I'd be interested to help. But, what do you intend to do that is not being done by that extension?

**Comment by hellerbarde** [comment]
I don't know. It was just an idea. If it can already do everything that boxmaker can do, then I can also just delete boxmaker :)

**Comment by roastedneutrons** [comment]
No. please don't. The beauty of boxmaker is its simplicity. I tried a few different ones. And I ported another one to inkscape 1.0 too. But i like the repo you are hosting, because it's the simplest and minimalistic. With everything else, I feel that there are too many options that bog you down. This repo serves a unique purpose, and I'd vote for it to stay.
